src/main/java/com/owncloud/android/lib/resources/files/UploadFileRemoteOperation.java

```diff
@@ -79,10 +80,27 @@ public UploadFileRemoteOperation(String localPath,
         this(localPath, remotePath, mimeType, requiredEtag, lastModificationTimestamp, null);
     }
 
+    public UploadFileRemoteOperation(String localPath,
+                                     String remotePath,
+                                     String mimeType,
+                                     String requiredEtag,
+                                     String lastModificationTimestamp,
+									 boolean disableRetries) {
```

Comment:
Somehow there are too many tabs, or?

Reply:
The formatting in this library is a bit of a mess (mix of tabs and spaces) and I didn't want to auto-format everything. The above parameters seem to use spaces instead of tabs. I moved the new one to spaces as well.